frontend/ui/elements/font_ui_fallbacks.lua

```diff
@@ -50,13 +50,13 @@ local genFallbackCandidates = function()
     end
 end
 
-local more_info_text = _([[
-If some book titles, dictionary entries and such are not displayed well but shown as ￾￾ or ��, it may be necessary to download the required fonts for those languages. They can then be enabled as additional UI fallback fonts.
+local more_info_text = T(_([[
+If some book titles, dictionary entries and such are not displayed well but shown as %1 or %2, it may be necessary to download the required fonts for those languages. They can then be enabled as additional UI fallback fonts.
 Fonts for many languages can be downloaded at:
 
 https://fonts.google.com/noto
 
-Only fonts named "Noto Sans xyz" or "Noto Sans xyz UI" (regular, not bold nor italic, not Serif) will be available in this menu. However, bold fonts will be used if their corresponding regular fonts exist.]])
+Only fonts named "Noto Sans xyz" or "Noto Sans xyz UI" (regular, not bold nor italic, not Serif) will be available in this menu. However, bold fonts will be used if their corresponding regular fonts exist.]]), "￾￾", "��")
```

Comment:
PS This GH preview doesn't show them.

![image](https://github.com/koreader/koreader/assets/202757/7ba2867f-a9ea-4dc3-a5b9-63470dc520b3)